src/api/app/controllers/webui/request_controller.rb

```diff
@@ -34,6 +34,9 @@ def show
       @open_reviews_for_staging_projects = @bs_request.reviews.opened.for_staging_projects
       @refresh = @action[:diff_not_cached]
 
+      # Collecting all issues in a hash. Each key is the issue name and the value is a hash containing all the issue details.
+      @issues = @action.fetch(:sourcediff, []).reduce({}) { |accumulator, sourcediff| accumulator.merge(sourcediff['issues']) }
```

Comment:
To test this, I ended up taking some data from production. I put this line above `@issues`:

```ruby
      @action[:sourcediff][0]['issues'] = {
        "CVE-2009-0067"=>
        {:created_at=> "Fri, 03 Jun 2016 07:21:20.000000000 UTC +00:00",
         :name=>"2009-0067",
         :tracker=>"cve",
         :label=>"CVE-2009-0067",
         :url=>"http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2009-0067",
         :updated_at=>"Fri, 03 Jun 2016 07:21:20.000000000 UTC +00:00",
         :summary=>
        "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem.  When the candidate has been publicized, the details for this candidate will be provided."},
          "CVE-2012-123465"=>
        {:created_at=>"Mon, 14 Nov 2022 09:25:19.000000000 UTC +00:00",
         :name=>"2012-123465",
         :tracker=>"cve",
         :label=>"CVE-2012-123465",
         :url=>"http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2012-123465",
         :updated_at=>"Mon, 14 Nov 2022 09:25:19.680796000 UTC +00:00"},
         "boo#123"=>{:created_at=>"Fri, 15 Oct 2004 15:34:00.000000000 UTC +00:00", :name=>"123", :tracker=>"bnc", :label=>"boo#123", :url=>"https://bugzilla.opensuse.org/show_bug.cgi?id=123", :state=>"CLOSED", :summary=>"Bugzilla displays MST"},
         "jsc#SLE-1234"=>
        {:created_at=>"Mon, 14 Nov 2022 09:25:20.000000000 UTC +00:00", :name=>"SLE-1234", :tracker=>"jsc", :label=>"jsc#SLE-1234", :url=>"https://jira.suse.com/browse/SLE-1234", :updated_at=>"Mon, 14 Nov 2022 09:25:20.782069000 UTC +00:00"}
      }
```